.github/workflows/ci.yml

```diff
@@ -47,13 +47,15 @@ jobs:
       - name: install bats
         shell: bash
         run: |
+	   export RAMALAMA_CONTAINER_ENGINE=docker
```

Comment:
GitHub has highlighted the formatting is not quite correct here. Might be a tabs/spaces thing. We've had similar situations before and it's ended up turning off our builds.